examples/apache-directory-list-local/webdav.conf

```diff
@@ -1,23 +1,16 @@
-ServerName webdav.server.com
-DocumentRoot /srv/webdav/root
+<VirtualHost *:80>
+    ServerName localhost
+    DocumentRoot /var/www/html/
 
-# This prevents indexes from being parsed (unless they're called the below...)
-AccessFileName .file-that-will-never-exist
-DirectoryIndex .file-that-will-never-exist
 
-# assuming webdav-js is checked out to /srv/webdav-js
-Alias /webdav-js /srv/web/webdav-js
+    # assuming webdav-js is checked out to /var/www/webdav-js
+    Alias /webdav-js /var/www/webdav-js
 
-<Location />
-    DAV on
+    <Location />
+        DirectoryIndex disabled
+        DAV on
 
-    Options +Indexes
-    HeaderName /webdav-js/examples/apache-directory-list-local/header.html
-
-    # the below only needed on a server configured with PHP
```

Comment:
I know it's not necessary for the Docker test, but I feel this is useful for someone setting up a WevDAV server. A lot of servers do have PHP enabled and this might catch people out. Open to discussion around this though! What do you think?

Reply:
After using `DirectoryIndex disabled` also disables `index.php`

Or am I misunderstanding something

Reply:
So, without those lines, if Apache is configured with PHP, when you request a file that ends `.php`, it gets parsed so `<?php print("test"); ?>` would result in just `test`, but it's not necessary for the Docker validation, it was more kept as an example if someone is using this as a basis for setting up on their server. Personally I think it'd be useful, but I'm open to arguments!

Reply:
I beleve you are talking about:
> ```
>    RemoveType .php
>    RemoveHandler .php
>    AddType text/plain .php
> ```

https://httpd.apache.org/docs/current/mod/mod_mime.html#addhandler

I will add that back in.

Reply:
Right indeed for security a webdav location is indeed best secured against php (or other server-side scripts) I'm disabling `.htaccess` too for good measure.

..working on it

Reply:
using `SetHandler None` work just as well without requiring language specific options

Reply:
done https://github.com/dom111/webdav-js/pull/21/commits/a6dc7987df17c271e15ff642586c06acf03f331c

Reply:
I think it's best to add php7.0 to the Dockerfile :question: :+1: 

```diff
--- RUN apt-get update -y && apt-get install -y apache2
+++ RUN apt-get update -y && apt-get install -y apache2 php7.0
```

Reply:
https://github.com/dom111/webdav-js/pull/21/commits/cc743e573b055f87b69e3b5a4619bc0bf0260bbc done